src/sqs/index.ts

```diff
@@ -101,6 +101,11 @@ export type ErrorHandlingResult = {
 	 */
 	retryable: boolean;
 
+	/**
+	 * Indicates if it was a failure
+	 */
+	isError: boolean;
```

Comment:
I find this one very confusing. What does "isError" means... This is a ErrorHandlingResult, means that there was a error in ErrorHandling, right? So I'd expect it to be true in some "catch" clause and false everywhere else... but then I see it is populated as true in quite some places... Could you please clarify?

Reply:
There can be 2 different cases:
1) The exception we got due to a faulty situation, like 500 from GitHub or`Cannot read property '<prop-name>' of undefined`
2) The situation where exception was expected and despite of it we can consider that the message is processed (retryable: false). For example: 401 from GitHub

For case 1) we set isError = true, for 2) we set isError = false.

Then we use isError to figure out if we need to send a metric that we have failed to process a webhook.  


So  to remove some  confusion I can rename it to , for example, isMessageFailed and add comment. Good?

Reply:
Changed the name and the comment + some logic around the non-failure results processing